app/models/config.rb

```diff
@@ -10,15 +10,16 @@ class Config
   CONFIG_FIELDS = [
     :insurance, :external_pledge_source, :pledge_limit_help_text,
     :language, :resources_url, :fax_service, :referred_by,
-    :practical_support
+    :practical_support, :start_of_week
   ].freeze
 
   # Define overrides for particular config fields.
   # Useful if there is no `_options` method.
   HELP_TEXT_OVERRIDES = {
     resources_url: 'A link to a Google Drive folder with CM resources. ' \
                    'Ex: https://drive.google.com/drive/my-resource-dir',
-    fax_service: 'A link to your fax service. ex: https://www.efax.com'
+    fax_service: 'A link to your fax service. ex: https://www.efax.com',
+    start_of_week: 'The start day of your budget week, if it doesn't start on Monday. ex: Sunday.
```

Comment:
lol @ bad advice from me

```suggestion
    start_of_week: "The start day of your budget week, if it doesn't start on Monday. ex: Sunday."
```